api/swagger.yml

```diff
@@ -198,6 +198,17 @@ components:
           type: string
           example: "main"
 
+    PathList:
+      type: object
+      required:
+        - paths
+      properties:
+        paths:
+          type: array
+          items:
```

Comment:
Not sure what "items" means.

Reply:
Swagger require you to describe each item in the array - the key of the array is 'paths' so each item is a path.
I'll add a description too.